src/client/headers/client/sqf/core.hpp

```diff
@@ -234,10 +231,172 @@ namespace intercept {
         bool is_equal_type_array(game_value left_array_, game_value right_array_);
         bool is_equal_type_params(game_value value_, game_value template_);
         int get_mission_version();
+    }  // namespace sqf
+}  // namespace intercept
+
+#include "client/pointers.hpp"
+
+namespace intercept {
+    namespace sqf {
+
+        /**
+         * We need other functions do be defined in header before we can template call functions
+         *  Otherwise we need forward declaration
+         */
+        template<typename T>
```

Comment:
```suggestion
        template<typename T = game_value>
```
such that we keep the old default behavior without having to update codebase